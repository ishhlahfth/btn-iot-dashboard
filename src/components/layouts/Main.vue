<template>
  <help-sidebar />
  <help-navbar />
  <div class="sm:ml-14 bg-grey-6">
    <router-view />
  </div>
</template>

<script>
import HelpNavbar from '@/components/molecules/Navbar.vue';
import HelpSidebar from '@/components/molecules/Sidebar.vue';

export default {
  name: 'Main',
  components: {
    HelpNavbar,
    HelpSidebar,
  },
};
</script>

<style></style>
