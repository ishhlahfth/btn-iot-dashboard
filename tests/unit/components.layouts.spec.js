import Main from '@/components/layouts/Main.vue';

describe('Main.vue', () => {
  it('should render correctly', () => {
    expect(Main.$el).toMatchSnapshot();
  });

  it('should render name Main', () => {
    expect(Main.name).toBe('Main');
  });
});
