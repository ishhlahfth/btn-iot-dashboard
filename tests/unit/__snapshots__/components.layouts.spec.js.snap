// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main.vue should render correctly 1`] = `undefined`;
